spring-server-generatedUpdated/src/main/java/io/swagger/api/BookApiController.java

```diff
@@ -37,33 +35,75 @@ public BookApiController(ObjectMapper objectMapper, HttpServletRequest request)
         this.request = request;
     }
 
-    public ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store" ,required=true )  @Valid @RequestBody Book body) {
+    public ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store", required = true) @Valid @RequestBody Book body) {
         String accept = request.getHeader("Accept");
-        return new ResponseEntity<Void>(HttpStatus.NOT_IMPLEMENTED);
+        System.out.println("Adding Book... " + body.getTitle());
```

Comment:
So instead of the System.out.println you should use your logger. You already have one added so I would change this line to

`log.info("Adding` book with title [{}]", body.getTitle());`